backend/cli.py

```diff
@@ -96,6 +97,25 @@ def import_gerrydb_view(layer: str, gpkg: str, replace: bool, rm: bool):
         ],
     )
 
+    # calculate layer extent to store in gerrydbtable
+    result = subprocess.run(
```

Comment:
H: Related to previous comments, I think we should add this as part of the `districtrmap` and not the `gerrydbtable`.